openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingTabs.tsx

```diff
@@ -31,7 +31,7 @@ const AtomicTestingTabs = ({ injectResultOverview }: Props) => {
   }
 
   return (
-    <Box mt={theme.spacing(2)}>
+    <Box mt={theme.spacing(4)}>
```

Comment:
This gives us a huge space, we would have to validate this part with @jborozco & @guillaumejparis